spec/views/hyrax/admin/collection_types/index.html.erb_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'spec_helper'
+
+RSpec.describe 'hyrax/admin/collection_types/index.html.erb', type: :view do
+  before do
+    assign(:collection_types, [
+             FactoryGirl.create(:collection_type, title: 'Test Title 1', machine_id: 'test_title_1'),
```

Comment:
@elrayle I think your previous comments were erased when I squashed commits, but just pointing out we updated this, and in fact we do need `FactoryGirl.create` here instead of build for the test to pass.